docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>For this function, I'd prefer there to be a docstring, typehints, and all the imports needed (even if you're re-importing) in this cell, so you can just copy-paste the whole cell into your workflow. </p><p> </p><p>Inside the function, I think using <code>os.scandir</code> is a little more complicated than necessary. I'd consider doing:</p><pre class="ql-syntax" spellcheck="false">files_for_mintpy = ['_water_mask.tif', '_corr.tif', '_unw_phase.tif', '_dem.tif', '_lv_theta.tif', '_lv_phi.tif']

for extension in files_for_mintpy:
    for file in data_dir.rglob(f'*{extension}'):
        dst_file = file.parent / f'{file.stem}_clipped{file.suffix}'
        gdal.Translate(destName=str(dst_file), srcDS=str(file), projWin=overlap)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>